SourceX/storm/storm.cpp

```diff
@@ -55,10 +56,14 @@ void GetBasePath(char *buffer, size_t size)
 
 	snprintf(buffer, size, "%s", path);
 	SDL_free(path);
+#else
+	snprintf(buffer, size, "%s", "D:\\");

```

Comment:
@marty28 this should be moved to the SDL2 implementation.

Reply:
I'm currently using SDL1.2, we also have the port of SDL2 I did for the MS Xbox SDK recently that we could use but it's slower and not as mature as 1.2 on Xbox.

I have not tested it with SDL2 since DevilutionX started supporting 1.2. I need to test it again, it might just work.

Maybe I should move this into SDL_GetBasePath?

Reply:
Ok, that explains some things. Using SDL1.2 makes sens. Yeah this should be moved to the our SDL1.2 SDL_GetBasePath compat.